pkg/reconciler/ingress/config/istio_test.go

```diff
@@ -308,3 +308,58 @@ func TestVSStatusEnabled(t *testing.T) {
 		})
 	}
 }
+
+func TestPodAddressabilityEnabled(t *testing.T) {
+	vsStatusTests := []struct {
```

Comment:
copy pasta-ed variable name from the other test, should probably update it

Reply:
Thanks!